docs/development/intro.rst

```diff
@@ -53,23 +53,27 @@ Four solutions exist:
 
 1. You add this message by hand into each of your commit messages (not recommended)
 
-2. You can sign each of your commits with the command: "``git commit -s``".
+2. You can sign each of your commits with the command: "``git commit -S``".
```

Comment:
We want people to signoff not to sign with gpg keys right?
This would be changing the rules defined in the PIG.